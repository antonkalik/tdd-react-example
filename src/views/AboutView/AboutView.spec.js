import { render, screen } from "@testing-library/react";
import { AboutView } from "./index";

describe("<AboutView />", () => {
  it("should render view", () => {
    render(<AboutView />);
    expect(screen.getByTestId("about-view")).toMatchSnapshot();
    expect(screen.getByTestId("about-view")).toHaveTextContent("About View");
  });
});
