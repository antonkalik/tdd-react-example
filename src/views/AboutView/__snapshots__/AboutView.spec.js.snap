// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutView /> should render view 1`] = `
<div
  data-testid="about-view"
>
  <h1>
    About View
  </h1>
</div>
`;
